app/views/spree/shared/_cart_box.html.erb

```diff
@@ -0,0 +1,216 @@
+<%= render partial: 'spree/shared/error_messages', locals: { target: @order } %>
+<% order_form&.object&.line_items.each do |line_item| %>
+  <%= order_form&.fields_for :line_items, line_item do |item_form| %>
+    <li class="cart-item line-item cart-line-item-container-<%= line_item.id %> shopping-cart-item " data-lineitem-id= "<%= line_item.id %>">
+      <button type="button" class="btn btn-primary remove-btn popup-cart-delete js-line-item-delete-btn-<%= line_item.id %> " data-line-item-id="<%= line_item.id %>" >Close</button>
+      <% if line_item&.variant&.images.length == 0 %>
```

Comment:
Please use spree default method to display image. Take example an example form cart page.  In this scenario, we are missing the no image edge case

Reply:
We will handle it later as instructed by @yaseeniqbal575 bhai